package.json

```diff
@@ -14,9 +14,12 @@
   "dependencies": {
     "@comunica/actor-init-sparql": "1.x.x",
     "@comunica/actor-init-sparql-rdfjs": "1.x.x",
+    "@comunica/bus-query-operation": "^1.13.0",
+    "@comunica/core": "^1.13.0",
     "@hapi/hoek": "9.x.x",
     "@hapi/joi": "17.x.x",
     "@oclif/command": "1.x.x",
+    "@rdfjs/data-model": "^1.1.2",
```

Comment:
Is this related to the gts upgrade?

Reply:
Yes, it does - `@rdfjs/data-model` is used in the code but wasn't declared here, which gts forbids.